update.js

```diff
@@ -74,16 +74,26 @@ module.exports = function update () {
 
     const dependency = extractDependency(pkg, config.branchPrefix, info.branchName)
 
-    if (!dependency) {
+    if (!dependency || dependency.length === 0) {
```

Comment:
this can now be an object OR an array.  Technically it would be 'depenedencies' and maybe it should always be an array..